lang/cs/Org.Apache.REEF.Client/Common/JobSubmissionResult.cs

```diff
@@ -48,7 +55,17 @@ internal abstract class JobSubmissionResult : IJobSubmissionResult
         private readonly HttpClient _client;
         private readonly IREEFClient _reefClient;
 
-        internal JobSubmissionResult(IREEFClient reefClient, string filePath)
+        /// <summary>
+        /// Number of retries when connecting to the Driver's HTTP endpoint.
+        /// </summary>
+        private readonly int _numberOfRetries;
+
+        /// <summary>
+        /// Retry interval in ms when connecting to the Driver's HTTP endpoint.
+        /// </summary>
+        private readonly int _retryInterval;
```

Comment:
Make it of type `TimeSpan` so there is no need for conversion on every invocation of `FetchFirstDriverStatus()`

Reply:
Done.